lib/commands/gesture.js

```diff
@@ -27,6 +27,8 @@ commands.performTouch = async function (gestures) {
     return await this.handleTap(gestures[0]);
   } else if (gestures.length === 1 && gestures[0].action === 'doubleTap') {
     return await this.handleDoubleTap(gestures[0]);
+  } else if (gestures.length === 1 && gestures[0].action === 'twoFingerTap') {
```

Comment:
This does not exist as a thing in any of the clients.

Two finger tap would be a particular version of multi touch, so needs to be decoded within `performMultiAction`.